pygfx/renderers/wgpu/_renderer.py

```diff
@@ -199,9 +200,9 @@ def pixel_ratio(self):
         """The ratio between the number of internal pixels versus the logical pixels on the canvas.
 
         This can be used to configure the size of the render texture
-        relative to the canvas' logical size. By default (value is None) the
-        used pixel ratio follows the screens pixel ratio on high-res
-        displays, and is 2 otherwise.
+        relative to the canvas' logical size. Can be set to None to set the default.
+        By default the pixel_ratio follows the screens pixel ratio on high-res
+        displays, and is 2.0 otherwise.
```

Comment:
I don't understand why you chose 2.0 as the default?

Reply:
It's best to have an integer number so the pixel grids align. With 1 there is no aa, with 3 the memory consumption might become a thing. Most high-res displays have a factor 2 as well.